Podfile

```diff
@@ -1,20 +1,27 @@
 source 'https://github.com/CocoaPods/Specs'
 
 workspace 'VideoQuickStart'
-platform :ios, '8.1'
 
 abstract_target 'TwilioVideo' do
   pod 'TwilioVideo', '~> 1.3.3'
 
+  target 'ARKitExample' do
+    platform :ios, '11.0'
+    project 'ARKitExample.xcproject'
+  end
+
   target 'VideoQuickStart' do
+    platform :ios, '8.1'
     project 'VideoQuickStart.xcproject'
   end
   
   target 'VideoCallKitQuickStart' do
+    platform :ios, '10.0'
     project 'VideoCallKitQuickStart.xcproject'
   end
 
   target 'ScreenCapturerExample' do
+    platform :ios, '8.1'
```

Comment:
Specifying the platform at the target level seemed like the clearest thing to do.